docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:alpine3.13 as jekyll
+
+RUN apk add --no-cache build-base gcc bash cmake git
+
+RUN gem install bundler -v "~>2.2.11" && gem install jekyll
```

Comment:
Where is Ruby coming from? Parent file system? This container should also fetch and install a recent version of Ruby.

Reply:
I did try to use multistage builds for dividing the ruby (and additional dependencies) and jekyll into two different stages. The Ruby was coming from `ruby:alpine3.13 ` but now it is part of jekyll image: jekyll/jekyll:4.2.0 (latest version of jekyll . Ruby version is bound ([`2.7.0`](https://github.com/envygeeks/jekyll-docker/blob/master/.ruby-version)) to the jekyll docker.